projects/angular/src/accordion/stepper/providers/stepper.service.spec.ts

```diff
@@ -70,4 +70,14 @@ describe('StepperService', () => {
     stepperService.navigateToNextPanel('1', true);
     expect(activeStepId).toBe('1');
   });
+
+  it('navigate to previous step', () => {
+    stepperService.navigateToNextPanel(panel1Id, true);
+    stepperService.navigateToPreviousPanel(panel2Id);
+
+    stepperService
+      .getPanelChanges(panel1Id)
+      .pipe(take(1))
```

Comment:
```suggestion
      .pipe(first())
```

This should actually be `first()` so that the test will fail if the `Observable` doesn't emit any value.

Reply:
It's actually a more complicated than just replacing the operator.